src/tests/stdlib/replace.ab

```diff
@@ -2,7 +2,11 @@ import * from "std/text"
 
 // Output
 // apple apple
+// apple
+// banana\\
 
 main {
     echo replace("banana banana", "banana", "apple")
+    echo replace("\\", "\\", "apple")
```

Comment:
To confirm that `replace` replaces multiple instances, may I suggest testing multiple occurrences of the search text:
```
echo replace("\\ \\", "\\", "apple")
echo replace("\\banana\\", "\\", "\\\\")
```

Reply:
You might also consider cloning the modified test, and overwriting replace_once.ab, with adjusted expected output.

Reply:
Added tests for replace_once and edited text.ab to make the test pass, though I did not know why.

Reply:
I'm still not entirely happy with these tests:
1. I had in mind that both test files would contain calls to `replace_once` and `replace`, with identical parameters, to show the differences in behaviour.
2. Each function call should test a specific use case; they don't all have to contain backslashes!
3. Each function call should contain multiple copies of the search text.

Reply:
For example:
```
import * from "std/text"

// Output
// TWO one one
// a\\b\c\d

main {
    echo replace("one one one", "one", "TWO")
    echo replace("a\\b\\c\\d", "\\", "\\\\")
}
```

Reply:
Thank you. I've also added a test `echo replace("mono\ndo\ntri", "do\n", "di\n")`, to check whether newlines are replaced with ` `(a space) unintentionally.